lib/gcp/rules.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+var api = require('../api');
+var logger = require('../logger');
+var utils = require('../utils');
+
+function _handleErrorResponse(response) {
+  if (response.body && response.body.error) {
+    return utils.reject(response.body.error, {code: 2});
+  }
+
+  logger.debug('[rules] error:', response.status, response.body);
+  return utils.reject('Unexpected error encountered deploying rules.', {code: 2});
+}
+
+/**
+ * Creates a new ruleset which can then be associated with a release.
+ * @param {String} projectId Project on which you want to create the ruleset.
+ * @param {Array} files Array of `{name, content}` for the source files.
+ */
+function createRuleset(projectId, files) {
+  var payload = {source: {files: files}};
+
+  return api.request('POST', '/v1/projects/' + projectId + '/rulesets', {
```

Comment:
I prefer to put the version # at the top of the file, so if we need to update API version in the future, it's a simpler change. (See https://github.com/firebase/firebase-tools/blob/master/lib/gcp/cloudfunctions.js#L10)

Reply:
Done.